### DISCUSSION TITLE
Contribution open source

### DISCUSSION THREAD

**Discussion by rami-pro** [fairnesscoop/permacoop#446] [General]
Bonjour,

Nous sommes un groupe de trois étudiants en Master MIAGE à l'Université de Paris-Saclay, et dans le cadre de notre formation, nous sommes encouragés à contribuer à des projets open source. Après avoir consulté le vôtre, nous avons constaté qu'il répond à nos critères : une bonne qualité de code, un projet actif, et, en plus, un projet français.

Nous serions ravis de pouvoir apporter notre contribution en matière de développement, tests, CI/CD, amélioration de la documentation pour rendre le projet plus accessible, ainsi qu'en participant à des sessions de pair programming avec l'un de vos ingénieurs. Si vous avez des issues ouvertes, nous serions également heureux que vous puissiez nous en attribuer afin de nous permettre de démarrer rapidement.

**Comment by florimondmanca** [comment]
Bonjour @rami-pro,

C'est très sympa de vous proposer pour nous donner un coup de main.

Concernant ce projet (Permacoop), il développé au compte-goutte en fonction des besoins internes de la coopérative, donc pas évident de pointer vers telle ou telle contribution prête à être réalisée.

Nous intervenons aussi sur des projets open source avec le service public, notamment [DiaLog](https://github.com/mtes-mct/dialog), qui est très actif et où l'équipe pourrait peut-être proposer quelques tickets non-bloquants pour vous permettre de contribuer. N'hésitez pas à y ouvrir une discussion.

À bientôt!
